.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*'
+  pull_request:
+  workflow_dispatch:
+
+env:
+  GRADLE_OPTS: >-
+    -Dorg.gradle.project.eu.xenit.cloudsmith.username=${{ secrets.CLOUDSMITH_USER }}
+    -Dorg.gradle.project.eu.xenit.cloudsmith.password=${{ secrets.CLOUDSMITH_APIKEY }}
+    -Dorg.gradle.project.org.alfresco.maven.nexus.username=${{ secrets.ALFRESCO_NEXUS_USERNAME }}
+    -Dorg.gradle.project.org.alfresco.maven.nexus.password=${{ secrets.ALFRESCO_NEXUS_PASSWORD }}
+
+jobs:
+  build-Test-Publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: temurin
+      - name: Login to Xenit Harbor docker registry
+        uses: docker/login-action@v2
+        with:
+          registry: hub.xenit.eu
+          username: ${{ secrets.XENIT_DOCKER_REGISTRY_USERNAME }}
+          password: ${{ secrets.XENIT_DOCKER_REGISTRY_PASSWORD }}
+      - name: Generate build number
+        id: buildNmbr-gen
+        run: echo "buildnumber="$(date +%Y%m%d%H%M) >> $GITHUB_OUTPUT
```

Comment:
This `BuildNumber` should be set in the gradle build (as a default), instead of being passed in every gradle call.